10-Sprint-Mission/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>판다마켓</title>
+</head>
+<body>
+  <header class="header">
+    <a href="index.html">
+      <img src="./img/lg/Property 1=md.png" alt="로그인">
+    </a>
+    <a href="./links/login.html">
+      <img src="./img/btn/btn_small.png" alt="로그인버튼">
+    </a>
+  </header>
+  <div>
+    <section class="top_banner">
+      
+        <button class="btn top">
+          <p>일상의 모든 물건을 <br>거래해보세요</p>
+          <a href="./links/item.html">
+          <img class="itemlink" src="./img/btn/state=active.png" alt="구경하러가기">
+          </a>
+        </button>
```

Comment:
P1:
button안에 a 태그가 들어가야할 이유가 있을까요?
페이지 이동을 시켜야하는 것이니 아래처럼 작성해도 될 것 같습니다.

```suggestion
        <div class="btn top">
          <p>일상의 모든 물건을 <br>거래해보세요</p>
          <a href="./links/item.html">
          <img class="itemlink" src="./img/btn/state=active.png" alt="구경하러가기">
          </a>
        </div>
```